<!DOCTYPE html
PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<html>
<head>
<title>BRESEQ :: Marginal Predictions</title>
<style type = "text/css">
body {font-family: sans-serif; font-size: 11pt;}
th {background-color: rgb(0,0,0); color: rgb(255,255,255);}
table {background-color: rgb(1,0,0); color: rgb(0,0,0);}
tr {background-color: rgb(255,255,255);}
.mutation_in_codon {color:red; text-decoration : underline;}
.snp_type_synonymous{color:green;}
.snp_type_nonsynonymous{color:blue;}
.snp_type_nonsense{color:red;}
.mutation_header_row {background-color: rgb(0,130,0);}
.read_alignment_header_row {background-color: rgb(255,0,0);}
.missing_coverage_header_row {background-color: rgb(0,100,100);}
.new_junction_header_row {background-color: rgb(0,0,155);}
.copy_number_header_row {background-color: rgb(153,102,0);}
.alternate_table_row_0 {background-color: rgb(255,255,255);}
.alternate_table_row_1 {background-color: rgb(235,235,235);}
.gray_table_row {background-color: rgb(230,230,245);}
.polymorphism_table_row {background-color: rgb(160,255,160);}
.highlight_table_row {background-color: rgb(192,255,255);}
.reject_table_row {background-color: rgb(255,200,165);}
.user_defined_table_row {background-color: rgb(255,255,0);}
.information_table_row {background-color: rgb(200,255,255);}
.junction_repeat {background-color: rgb(255,165,0)}
.junction_gene {}
.hidden { display: none; }
.unhidden { display: block; }

</style>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<script type="text/javascript">
  function hideTog(divID) {
    var item = document.getElementById(divID);
    if (item) {
      item.className=(item.className=='hidden')?'unhidden':'hidden';
    }
  }
  function showTog(butID) {
    var button = document.getElementById(butID);
    if (button) {
      button.value=(button.value=='Show')?'Hide':'Show';
    }
  }
</script>

</head>
<body>
<table width="100%" border="0" cellspacing="0" cellpadding="3">
<tr>
<td><a href="http://barricklab.org/breseq"><img src="evidence/breseq_small.png" /></a></td>
<td width="100%">
<b><i>breseq</i></b>&nbsp;&nbsp;version 0.36.0&nbsp;&nbsp;
<br><a href="index.html">mutation predictions</a> | 
<a href="marginal.html">marginal predictions</a> | 
<a href="summary.html">summary statistics</a> | 
<a href="output.gd">genome diff</a> | 
<a href="log.txt">command line log</a>
</td></tr></table>

<p>
<p>
<table border="0" cellspacing="1", cellpadding="3">
<tr><th colspan="12" align="left" class="read_alignment_header_row">Marginal read alignment evidence (highest frequency 20 of 31 shown, sorted by frequency from high to low)</th></tr>
<tr>
<th>&nbsp;</th>
<th>seq&nbsp;id</th>
<th colspan="2">position</th>
<th>ref</th>
<th>new</th>
<th>freq</th>
<th>score&nbsp;(cons/poly)</th>
<th>reads</th>
<th>annotation</th>
<th>genes</th>
<th width="100%">product</th>
</tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_109.html">*</a></td><td align="center">NZ_CP064350</td><td align="right">2,843,762</td><td align="right">0</td><td align="center">G</td><td align="center">A</td><td align="right">59.3%
</td><td align="center">11.5
/&nbsp;247.1
</td><td align="center">177</td><td align="center">noncoding&nbsp;(43/76&nbsp;nt)</td><td align="center"><i>IT766_RS12985</i></td><td align="left">tRNA&#8209;Val</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_58.html">*</a></td><td align="center">NZ_CP064350</td><td align="right">77,917</td><td align="right">0</td><td align="center">C</td><td align="center">T</td><td align="right">58.7%
</td><td align="center">80.4
/&nbsp;202.6
</td><td align="center">143</td><td align="center">intergenic&nbsp;(+310/&#8209;227)</td><td align="center"><i>IT766_RS00445/IT766_RS00450</i></td><td align="left">phosphoadenylyl&#8209;sulfate reductase/16S ribosomal RNA</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_60.html">*</a></td><td align="center">NZ_CP064350</td><td align="right">77,966</td><td align="right">0</td><td align="center">G</td><td align="center">A</td><td align="right">52.5%
</td><td align="center">20.5
/&nbsp;269.2
</td><td align="center">160</td><td align="center">intergenic&nbsp;(+359/&#8209;178)</td><td align="center"><i>IT766_RS00445/IT766_RS00450</i></td><td align="left">phosphoadenylyl&#8209;sulfate reductase/16S ribosomal RNA</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_59.html">*</a></td><td align="center">NZ_CP064350</td><td align="right">77,954</td><td align="right">0</td><td align="center">T</td><td align="center">C</td><td align="right">50.6%
</td><td align="center">4.9
/&nbsp;296.4
</td><td align="center">166</td><td align="center">intergenic&nbsp;(+347/&#8209;190)</td><td align="center"><i>IT766_RS00445/IT766_RS00450</i></td><td align="left">phosphoadenylyl&#8209;sulfate reductase/16S ribosomal RNA</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_97.html">*</a></td><td align="center">NZ_CP064350</td><td align="right">2,279,524</td><td align="right">0</td><td align="center">T</td><td align="center">G</td><td align="right">38.8%
</td><td align="center">67.3
/&nbsp;21.2
</td><td align="center">50</td><td align="center">intergenic&nbsp;(&#8209;384/&#8209;102)</td><td align="center"><i>IT766_RS10290/IT766_RS10295</i></td><td align="left">TSUP family transporter/retron St85 family RNA&#8209;directed DNA polymerase</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_66.html">*</a></td><td align="center">NZ_CP064350</td><td align="right">314,753</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">33.3%
</td><td align="center">183.3
/&nbsp;53.3
</td><td align="center">111</td><td align="center">pseudogene&nbsp;(1654/2291&nbsp;nt)</td><td align="center"><i>IT766_RS01600</i></td><td align="left">patatin&#8209;like phospholipase family protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_102.html">*</a></td><td align="center">NZ_CP064350</td><td align="right">2,425,035</td><td align="right">0</td><td align="center">T</td><td align="center">A</td><td align="right">32.5%
</td><td align="center">195.0
/&nbsp;37.9
</td><td align="center">114</td><td align="center"><font class="snp_type_nonsynonymous">I442F</font>&nbsp;(<font class="mutation_in_codon">A</font>TT&rarr;<font class="mutation_in_codon">T</font>TT)&nbsp;</td><td align="center"><i>rapA</i></td><td align="left">RNA polymerase&#8209;associated protein RapA</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_110.html">*</a></td><td align="center">NZ_CP064350</td><td align="right">2,877,632</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">32.0%
</td><td align="center">126.3
/&nbsp;25.6
</td><td align="center">78</td><td align="center"><font class="snp_type_nonsynonymous">R550T</font>&nbsp;(A<font class="mutation_in_codon">G</font>A&rarr;A<font class="mutation_in_codon">C</font>A)&nbsp;</td><td align="center"><i>IT766_RS13155</i></td><td align="left">hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_117.html">*</a></td><td align="center">NZ_CP064351</td><td align="right">293,005</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">31.5%
</td><td align="center">264.0
/&nbsp;55.8
</td><td align="center">143</td><td align="center"><font class="snp_type_synonymous">A91A</font>&nbsp;(GC<font class="mutation_in_codon">G</font>&rarr;GC<font class="mutation_in_codon">C</font>)&nbsp;&Dagger;</td><td align="center"><i>nadE</i></td><td align="left">ammonia&#8209;dependent NAD(+) synthetase</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_118.html">*</a></td><td align="center">NZ_CP064351</td><td align="right">293,007</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">28.0%
</td><td align="center">266.1
/&nbsp;51.4
</td><td align="center">132</td><td align="center"><font class="snp_type_nonsynonymous">A91P</font>&nbsp;(<font class="mutation_in_codon">G</font>CG&rarr;<font class="mutation_in_codon">C</font>CG)&nbsp;&Dagger;</td><td align="center"><i>nadE</i></td><td align="left">ammonia&#8209;dependent NAD(+) synthetase</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_75.html">*</a></td><td align="center">NZ_CP064350</td><td align="right">839,390</td><td align="right">0</td><td align="center">G</td><td align="center">C</td><td align="right">27.5%
</td><td align="center">194.7
/&nbsp;28.5
</td><td align="center">91</td><td align="center"><font class="snp_type_synonymous">T137T</font>&nbsp;(AC<font class="mutation_in_codon">C</font>&rarr;AC<font class="mutation_in_codon">G</font>)&nbsp;</td><td align="center"><i>IT766_RS03890</i></td><td align="left">phosphate/phosphite/phosphonate ABC transporter substrate&#8209;binding protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_73.html">*</a></td><td align="center">NZ_CP064350</td><td align="right">731,738</td><td align="right">0</td><td align="center">G</td><td align="center">C</td><td align="right">25.6%
</td><td align="center">287.8
/&nbsp;25.1
</td><td align="center">121</td><td align="center"><font class="snp_type_nonsynonymous">K574N</font>&nbsp;(AA<font class="mutation_in_codon">G</font>&rarr;AA<font class="mutation_in_codon">C</font>)&nbsp;</td><td align="center"><i>htpG</i></td><td align="left">molecular chaperone HtpG</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_125.html">*</a></td><td align="center">NZ_CP064351</td><td align="right">495,514</td><td align="right">0</td><td align="center">G</td><td align="center">C</td><td align="right">24.7%
</td><td align="center">182.9
/&nbsp;18.6
</td><td align="center">74</td><td align="center">intergenic&nbsp;(+41/&#8209;17)</td><td align="center"><i>tnpA/IT766_RS16355</i></td><td align="left">IS200/IS605&#8209;like element IS1004 family transposase/GNAT family N&#8209;acetyltransferase</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_124.html">*</a></td><td align="center">NZ_CP064351</td><td align="right">495,490</td><td align="right">0</td><td align="center">T</td><td align="center">G</td><td align="right">24.6%
</td><td align="center">129.1
/&nbsp;22.9
</td><td align="center">68</td><td align="center">intergenic&nbsp;(+17/&#8209;41)</td><td align="center"><i>tnpA/IT766_RS16355</i></td><td align="left">IS200/IS605&#8209;like element IS1004 family transposase/GNAT family N&#8209;acetyltransferase</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_93.html">*</a></td><td align="center">NZ_CP064350</td><td align="right">2,075,278</td><td align="right">0</td><td align="center">C</td><td align="center">A</td><td align="right">23.9%
</td><td align="center">320.6
/&nbsp;30.4
</td><td align="center">117</td><td align="center"><font class="snp_type_nonsynonymous">G39C</font>&nbsp;(<font class="mutation_in_codon">G</font>GT&rarr;<font class="mutation_in_codon">T</font>GT)&nbsp;</td><td align="center"><i>flgE</i></td><td align="left">flagellar hook protein FlgE</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_108.html">*</a></td><td align="center">NZ_CP064350</td><td align="right">2,841,732</td><td align="right">0</td><td align="center">G</td><td align="center">A</td><td align="right">23.8%
</td><td align="center">299.4
/&nbsp;91.7
</td><td align="center">126</td><td align="center"><font class="snp_type_synonymous">E41E</font>&nbsp;(GA<font class="mutation_in_codon">G</font>&rarr;GA<font class="mutation_in_codon">A</font>)&nbsp;</td><td align="center"><i>IT766_RS12965</i></td><td align="left">hypothetical protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_121.html">*</a></td><td align="center">NZ_CP064351</td><td align="right">312,541</td><td align="right">0</td><td align="center">T</td><td align="center">A</td><td align="right">23.2%
</td><td align="center">335.3
/&nbsp;23.2
</td><td align="center">151</td><td align="center"><font class="snp_type_nonsynonymous">*919Y</font>&nbsp;(TA<font class="mutation_in_codon">A</font>&rarr;TA<font class="mutation_in_codon">T</font>)&nbsp;</td><td align="center"><i>prtV</i></td><td align="left">M6 family metalloprotease PrtV</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_67.html">*</a></td><td align="center">NZ_CP064350</td><td align="right">314,756</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">22.7%
</td><td align="center">257.8
/&nbsp;25.0
</td><td align="center">110</td><td align="center">pseudogene&nbsp;(1657/2291&nbsp;nt)</td><td align="center"><i>IT766_RS01600</i></td><td align="left">patatin&#8209;like phospholipase family protein</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_78.html">*</a></td><td align="center">NZ_CP064350</td><td align="right">990,039</td><td align="right">0</td><td align="center">C</td><td align="center">G</td><td align="right">22.3%
</td><td align="center">364.7
/&nbsp;31.0
</td><td align="center">148</td><td align="center"><font class="snp_type_synonymous">T110T</font>&nbsp;(AC<font class="mutation_in_codon">G</font>&rarr;AC<font class="mutation_in_codon">C</font>)&nbsp;</td><td align="center"><i>sbcB</i></td><td align="left">exodeoxyribonuclease I</td></tr>
<tr class="polymorphism_table_row"><td><a href="evidence/RA_81.html">*</a></td><td align="center">NZ_CP064350</td><td align="right">1,376,105</td><td align="right">0</td><td align="center">C</td><td align="center">A</td><td align="right">21.8%
</td><td align="center">329.9
/&nbsp;24.1
</td><td align="center">110</td><td align="center"><font class="snp_type_synonymous">G84G</font>&nbsp;(GG<font class="mutation_in_codon">C</font>&rarr;GG<font class="mutation_in_codon">A</font>)&nbsp;</td><td align="center"><i>IT766_RS06230</i></td><td align="left">DUF2960 domain&#8209;containing protein</td></tr>
</table>

</body></html>